packages/sdk/lib/nats.ts

```diff
@@ -0,0 +1,39 @@
+import { connect, ConnectionOptions, NatsConnection } from 'nats';
+
+export async function createNatsConnection(config: ConnectionOptions): Promise<NatsConnection> {
+  if (!config) {
+    return;
+  }
```

Comment:
the `config` parameter is marked as required, but is not used anywhere.

it should be preferred over the env vars

Reply:
should be fixed now